packages/terra-layout/src/_LayoutSlidePanel.jsx

```diff
@@ -148,29 +141,14 @@ class LayoutSlidePanel extends React.Component {
       { 'is-animated': isAnimated && isOverlay && !!panelContent },
     ]);
 
-    const panel = (
-      <div className={panelClasses} aria-hidden={!isOpen ? 'true' : null} ref={this.setPanelNode}>
-        <HoverTarget
-          isContentHidden={this.isHidden}
-          contentRefCallback={this.setContentNode}
-          text={toggleText}
-          isOpen={isOpen || !isToggleEnabled}
-          hoverIsEnabled={!compactSize && isOverlay}
-          onHoverOff={() => { if (isOpen) { onToggle(); } }}
-          onHoverOn={() => { if (!isOpen) { onToggle(); } }}
-          onClick={onToggle}
-        >
-          {panelContent}
-        </HoverTarget>
-      </div>
-    );
-
     return (
       <div
         {...customProps}
         className={slidePanelClassNames}
       >
-        {panel}
+        <div className={panelClasses} aria-hidden={this.isHidden ? 'true' : 'false'} ref={this.setPanelNode}>
```

Comment:
do we want to keep the quotes around true and false? Looks like you removed them above on [line 101](https://github.com/cerner/terra-framework/pull/80/files#diff-89a6c6532ba24f2bd1193b25680c573aR101)

Reply:
For html attributes that use the word true or false, e.g. aria-hidden="true", we should pass that value to the attribute as a string instead of a boolean. This is something react 16 seems to be more particular about. In react 15, we could pass true or false as a boolean type to the attribute and it would convert it to a string for us when passed to html attributes, or maybe the prop validation didn't really care. React 16 seems to throw prop errors now for invalid types if we pass boolean values to html attributes that use true or false.

Reply:
Updated.